README.md

```diff
@@ -8,15 +8,14 @@ Use this action to track a deployment on [Render](https://render.com)
 
 ## Action Inputs
 
-| name         | description                                                                                              |
-| ------------ | -------------------------------------------------------------------------------------------------------- |
-| `email`      | User email used to sign-in to Render's API                                                               |
-| `password`   | User password used to sign-in to Render's API                                                            |
-| `service-id` | The id of the Render service to be tracked                                                               |
-| `token`      | GitHub token                                                                                             |
-| `sleep`      | Sleep time between the render deployment success and setting the Github deployment as successful         |
-| `retries`    | Maximum number of retries trying to find the deployment. Note: Retries will be attempted every 5 seconds |
-| `wait`       | Sleep time between retries to find Render deployments statuses                                           |
+| Name | Description | Required |
+| -----| ----------- | -------- |
+| `service-id`   | The id of the Render service to be tracked. | **Yes** ✅ |
+| `render-token` | Render API Token to use - [see documentation](https://render.com/docs/api#creating-an-api-key) | No ❌
+| `github-token` | GitHub Token to use | No ❌
+| `sleep`        | Sleep time between the render deployment success and setting the Github deployment as successful.<br/>*(default: 0)* | No ❌
+| `retries`      | Maximum number of retries trying to find the deployment. **Note:** Retries will be attempted every 5 seconds</br>*(default: 50)* | No ❌
+| `wait`         | Sleep time between retries to find Render deployments statuses<br/>*(default: 8000 [8 seconds])* | No ❌
```

Comment:
I think you're missing a `|` at the end of the rows

Reply:
Oops :eyes: updated :smile: